src/components/MemberProfileModal.vue

```diff
@@ -34,7 +34,7 @@
                 center
                 rounded="circle"
                 class="profile-image"
-                :src="`${Strings.get('directories.members', 'assets')}/${profile.id+'.jpg'}`"
```

Comment:
We need to alter this eventually so that the image URL is retrieved from the backend.